src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -716,6 +716,20 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "ExposureSubmissionTestresultAvailable_CloseAlertButtonContinue" = "Ergebnis anzeigen";
 
+
+/* SRS Consent Screen */
+
+"SRSConsentScreen_warnOthers" = "Andere warnen";
```

Comment:
Better to use heading1 here and heading2 for SRSConsentScreen_HelpOthers, if you are going generic?

Reply:
i dont undetrstand